src/test/cool/main.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+namespace HelloWorld
+{
+    class Hello
+    {
+        static void Main()
```

Comment:
ideally this would be a unit test with junit xml output for Azure Devops build pipelines to consume.